### PR TITLE
Task #200413:[AG Extended Form-Enrollment detail-summary page modific…

### PR DESCRIPTION
**Pull request by Dhanashree-Patil95**
…ation]

### CODE REVIEW

File: apps/front-end/src/pages/front-end/BenificiaryEnrollment.js [hunk 1]
```diff
@@ -159,8 +159,20 @@ export default function BenificiaryEnrollment() {
                   : "-"}
               </FrontEndTypo.H3>
             </HStack>
+            <HStack alignItems="Center" justifyContent="space-between">
```
Comment (sagarkoshti1990) [inline apps/front-end/src/pages/front-end/BenificiaryEnrollment.js:162]: Remove `justifyContent="space-between"`

File: apps/front-end/src/pages/front-end/BenificiaryEnrollment.js [hunk 2]
```diff
@@ -159,8 +159,20 @@ export default function BenificiaryEnrollment() {
...
+              <FrontEndTypo.H3 color="textGreyColor.50" flex="0.3" pb="2">
```
Comment (sagarkoshti1990) [inline apps/front-end/src/pages/front-end/BenificiaryEnrollment.js:163]: `flex="0.3"` change to `flex="3"`

File: apps/front-end/src/pages/front-end/BenificiaryEnrollment.js [hunk 3]
```diff
@@ -159,8 +159,20 @@ export default function BenificiaryEnrollment() {
...
+                {t("DATE_OF_ENROLLMENT")}
+              </FrontEndTypo.H3>
+              <FrontEndTypo.H3 color="textGreyColor.800" flex="0.4">
```
Comment (sagarkoshti1990) [inline apps/front-end/src/pages/front-end/BenificiaryEnrollment.js:166]: `flex="0.4"` change to `flex="4"`

**Review by sagarkoshti1990 — CHANGES_REQUESTED** [review]

File: apps/front-end/src/pages/front-end/BenificiaryEnrollment.js (hunk 1 above)
Comment (Dhanashree-Patil95) [reply apps/front-end/src/pages/front-end/BenificiaryEnrollment.js]: Resolved

File: apps/front-end/src/pages/front-end/BenificiaryEnrollment.js (hunk 2 above)
Comment (Dhanashree-Patil95) [reply apps/front-end/src/pages/front-end/BenificiaryEnrollment.js]: Resolved

File: apps/front-end/src/pages/front-end/BenificiaryEnrollment.js (hunk 3 above)
Comment (Dhanashree-Patil95) [reply apps/front-end/src/pages/front-end/BenificiaryEnrollment.js]: Resolved

**Review by sagarkoshti1990 — APPROVED** [review]

**Pull request merged**